sdk/core/core/src/az_span.c

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <az_span.h>
+
+#include <_az_cfg.h>
+
+/**
+ * ASCII lower case.
+ */
+AZ_NODISCARD AZ_INLINE az_result_byte az_ascii_lower(az_result_byte const value) {
+  return 'A' <= value && value <= 'Z' ? value + ('a' - 'A') : value;
```

Comment:
isn't it better to have 'a' - 'A' as a constant and then just use it here. Instead of calculating this every time ....